### DISCUSSION TITLE
blog/open-graph-images

### DISCUSSION THREAD

**Discussion by giscus** [SammyMusyoki/Portofolio-new#12] [Announcements]
# blog/open-graph-images

The open graph images serve as the preview thumbnail, giving Users a glimpse of the content before they decide to click on the link.

http://localhost:3000/blog/open-graph-images

**Comment by SammyMusyoki** [comment]
hi

**Comment by MusyokiSammy** [comment]
heey there

**Comment by lewis-otwori** [comment]
Nice blog bro

**Reply by SammyMusyoki** [reply]
Thanks bro
